digester_test.go

```diff
@@ -29,6 +29,8 @@ func (s *testSQLDigestSuite) TestNormalize(c *C) {
 		expect string
 	}{
 		{"SELECT 1", "select ?"},
+		{"select null", "select ?"},
+		{"select \\N", "select ?"},
```

Comment:
please add a test case for:

```sql
SELECT `null`
```

since `` `null` `` is a quoted identifier this should *not* be normalized to `select ?`.